notification-portlet-webapp/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -120,7 +120,20 @@
                 <name>usePortalJsLibs</name>
                 <value>false</value>
             </preference>
-
+            
+            <!-- Indicates whether the portlet will mark a notification as 'COMPLETE' when
```

Comment:
mark a notification as the configured 'StateChangeNotificationServiceDecorator.applyState' value